enhancements/cluster-logging/cluster-logging-es-rollover-data-design.md

```diff
@@ -0,0 +1,253 @@
+              max_age:   3d
+        delete:
+          min_age: 7d
+    templates:
+    - name:  infra
+      template:
+        index_patterns:
+        - node.infra*
+        settings:
+          number_of_shards: 1,        #value based on es node count policy
+          number_of_replicas: 1,      #value based on redundancy policy
+        aliases:
+        - infra  
+    seedings:
+    - prefix: node.infra              #creates node.infra-00001 aliased node.infra-write     
+      policyRef: infra-policy
+```
+#### Elasticsearch Operator
+The `elasticsearch-operator` will be modified to:
+* Expose the index management API
+* Create and seed specified index templates
+* Create the initial indices as needed
+* Block data ingestion if needed until initial index is seeded
+* Deploy curation CronJob for each entry in the specified seedings to rollover using the defined policy
+
+#### Collector 
+* Update the [`fluent-plugin-viaq-data-model`](https://github.com/ViaQ/fluent-plugin-viaq_data_model) to allow defining a static index to write logs
+
+Example configuration:
+```    
+  ....
+   <elasticsearch_index_name>
+     tag "**"
+     name_type static index_name=>'container.app-write'
+   </elasticsearch_index_name>
+```
+#### Deprecations
+* The Curator Cronjob deployed by the `cluster-logging-operator` will be removed. The responsibilities for curation will be subsumed by
+ implementation of Elasticsearch rollover management.
+* Curation configuration by namespace is no longer configurable and is restricted to cluster
+wide settings associated with log type 
+
+## Design Details
+
+### Test Plan
+* Regression tests will be executed to confirm no regressions from previous releases
+#### Unit Tests
+* Unit tests will be modified to account for the change in data design
+#### Integration Tests
+* e2e tests will be modified to account for the change in data design
+
+### Upgrade / Downgrade Strategy
+
+#### Upgrade
+The `elasticsearch-operator` will migrate existing indices to the new data design:
+* Indices beginning with `project.*` are indexed to `container.app-write`
+* Indices beginning with `.operations.*` are indexed to `journal.infra-write`
+* Migrated indices are deleted after migration
+
+**Note:** Migrated container infrastructure logs are available via the `infra` alias only and
+subsequently curated from the cluster during period rollover.
+
+The `cluster-logging-operator` will remove previously deployed CronJobs that were owned by the ClusterLogging instance
+
+#### Downgrade
+Downgrades should be discouraged unless we know for certain the Elasticsearch version managed by cluster logging is the same version.  There is risk that Elasticsearch may have migrated data that is unreadable by an older version.
+
+## Implementation History
+
+| release|Description|
+|---|---|
+|4.4| GA release of rollover data design
+
+## Drawbacks
+
+The drawback to note implementing this change is that Cluster Logging will:
```

Comment:
did you mean 

```suggestion
The drawback to not implementing this change is that Cluster Logging will:
```

?

but normally this section is for drawbacks associated w/ implementing it.